test/parallel/test-benchmark-querystring.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+require('../common');
+
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('querystring', ['n=1', 'input=', 'type=']);
```

Comment:
This would result in multiple undefined values so we should at least use a existing value.

Reply:
Thanks for the context; I was trying to replicate the pattern used in other benchmark invocations, where an empty string is used. I've pulled in what looked like the appropriate "no-op" values from the existing benchmark.